src/ServiceContainer/SymfonyExtension.php

```diff
@@ -266,4 +292,16 @@ private function autodiscoverBootstrap($bootstrap): ?string
 
         return is_string($bootstrap) ? $bootstrap : null;
     }
+
+    private function handleAutoConfiguration(ContainerBuilder $container): void
+    {
+        $container->registerForAutoconfiguration(Context::class)->addTag(self::CONTEXT_TAG);
```

Comment:
This probably won't work, since `$container` here is Behat's container, not the application's container with all the context services. There's a bundle inside this extension where it is already implemented.